backend/app/services/implementations/sign_in_logs_service.py

```diff
@@ -24,9 +25,12 @@ def __init__(self, logger):
     def create_sign_in_log(self, user_id):
         sign_in = {"id": user_id, "time": datetime.now()}
         try:
-            new_sign_in = SignInLogs(**sign_in)
-            db.session.add(new_sign_in)
-            db.session.commit()
+            print("!!!!LADBLNALINPIWGAEN")
```

Comment:
Lol are these print statements supposed to be here or were they for debugging

Reply:
+1 this should be deleted (`!!!!LADBLNALINPIWGAEN` is an interesting choice?)

Reply:
Oops missed that one... Thank you!